task/04-arrays-tasks.js

```diff
@@ -268,7 +271,7 @@ function getSecondItems(arr) {
  *  [ 1,2,3,4,5 ] => [ 1, 2,2, 3,3,3, 4,4,4,4, 5,5,5,5,5 ]
  */
 function propagateItemsByPositionIndex(arr) {
-   throw new Error('Not implemented');
+   return arr.reduce((acc, curr, i) => acc.concat(Array.from({length: i + 1}, () => curr)), []);
```

Comment:
Please consider using `flatMap`

Reply:
For some reason I forgot flatMap even existed :)